app/position/positionkit.cpp

```diff
@@ -127,7 +137,18 @@ AbstractPositionProvider *PositionKit::constructActiveProvider( AppSettings *app
   }
   else
   {
-    return constructProvider( QStringLiteral( "external" ), providerId );
+    // find name of the active provider
+    QString providerName;
+    QVariantList providers = appsettings->savedPositionProviders();
+    for ( const auto &provider : providers )
+    {
+      if ( provider.toList()[1] == providerId )
```

Comment:
is it worth doing a sanity check that there are two items in the list to avoid a crash in some rare circumstances?

Reply:
That array is always filled with two items, but true, one never knows what can happen in future :)